README.md

```diff
@@ -8,9 +8,12 @@
 git clone git@github.com:12-cactus/espi.git && cd espi
 npm install
 ```
+Create .env file with .env.example
 
-Then you have to make a tunnel and then start.
+With the Telegram's BotFather create `/newbot`
+then copy/paste the token on the .env file
```

Comment:
Create your dev bot using [BotFather](https://telegram.me/botfather) and the command `/newbot`, then add your bot token to the `.env` file.